lib/katello/engine.rb

```diff
@@ -87,15 +87,22 @@ def find_assets(args = {})
       require 'katello/apipie/validators'
     end
 
-    initializer "katello.register_actions", :before => 'foreman_tasks.initialize_dynflow' do |_app|
+    # make sure the Katello plugin is initialized before `after_initialize`
+    # hook so that the resumed Dynflow tasks can rely on everything ready.
+    initializer 'katello.register_plugin', :before => :finisher_hook do
```

Comment:
Changing them all to before finisher_hook still will rely on this being in the code first vs a way to perform this deterministic-ally before the other initializer actions?